src/conf.py

```diff
@@ -73,7 +93,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+exclude_patterns = [ '_build', 'cmake/blt/docs' ]
```

Comment:
This excludes the BLT docs that exist under this directory but aren't linked in to the toctree.